src/index.js

```diff
@@ -17,7 +17,11 @@ Sentry.init({
 
 const queryClient = new QueryClient({
   defaultOptions: {
+    mutations: {
```

Comment:
Tilsvarer v3 (https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#queries-and-mutations-per-default-need-network-connection-to-run)